src/graphics/vulkan.zig

```diff
@@ -195,11 +206,26 @@ pub fn createInstance() void {
 		std.log.debug("\t{s}", .{@as([*:0]const u8, @ptrCast(&ext.extensionName))});
 	}
 
+	var createFlags: u32 = 0;
+	var extensions = std.ArrayList([*c]const u8).initCapacity(main.stackAllocator.allocator, glfwExtensionCount) catch unreachable;
```

Comment:
I think initCapcity is mislead here, and might actually result in doing more allocations.